.github/ci-test-connections.yaml

```diff
@@ -29,7 +29,7 @@ connections:
     port: 443
     login: $SNOWFLAKE_ACCOUNT_NAME
     password: $SNOWFLAKE_PASSWORD
-    schema: ASTROFLOW_CI
+    schema: ASTROFLOW_TMP_CI
```

Comment:
Remove this schema before merging it back to main.

Reply:
Removed this